tests/unit/test_chain_tezos.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from aleph_client.chains.common import delete_private_key_file
+from aleph_client.chains.tezos import TezosAccount, get_fallback_account
+from dataclasses import dataclass, asdict
+
+@dataclass
+class Message:
+    chain: str
+    sender: str
+    type: str
+    item_hash: str
+
+
+def test_get_fallback_account():
+    delete_private_key_file()
+    account: TezosAccount = get_fallback_account()
+
+    assert account.CHAIN == "TEZOS"
+    assert account.CURVE == "secp256k1"
+    assert account._account.public_key()
+
+@pytest.mark.asyncio
+async def test_TezosAccount():
```

Comment:
We use snake case for function names, so `test_tezos_account` is more idiomatic.